src/main/java/io/cloudsoft/jmeter/JMeterSshDriver.java

```diff
@@ -62,7 +62,7 @@ public void launch() {
     @Override
     public boolean isRunning() {
         StringBuilder command = new StringBuilder(getExpandedInstallDir())
-                .append("/bin/jmeter -v");
+                .append("/usr/bin/pgrep jmeter");
```

Comment:
I suppose this is more accurate! I'm concerned that it would mean that using the `pause` and `changeLoad` effectors would cause the entity's status to change to `ON_FIRE` since both stop the running process. 

What kind of machine is unable to run `jmeter -v`? If it can't do that can it even run jmeter?